backend/utils/parseText.js

```diff
@@ -0,0 +1,40 @@
+const natural = require('natural');
+const wordsToNumbers = require('words-to-numbers');
+
+exports.get_query_dict = (text) => {
```

Comment:
Can we remove the logs in this function, they might pollute the app logs.
However, we can maybe add something at the end like `console.log({text, dict})` for debugging purposes when the app is deployed.